neo/utils.py

```diff
@@ -490,10 +491,11 @@ def cut_block_by_epochs(block, properties=None, reset_time=False):
         for epoch in epochs:
             new_segments = cut_segment_by_epoch(
                 seg, epoch=epoch, reset_time=reset_time)
-            block.segments += new_segments
```

Comment:
is there a reason to use `+=` rather than `extend` here?

Reply:
I am anticipating the day wher we will have pseudo list for relationship.
I guess append and extend will be part of methods not sure for +=

Reply:
then it would be better to stick to `extend`, no?